plugin/trino-http-event-listener/src/main/java/io/trino/plugin/httpquery/HttpEventListener.java

```diff
@@ -126,6 +126,7 @@ public void onSuccess(StatusResponse result)
                     }
                     catch (InterruptedException ignored) {
                         log.warn("Sleep between send attempts was interrupted, aborting send");
+                        return;
```

Comment:
Will be probably gone in final shape.
But in case it is not there should be also `Thread.currentThread().interrupt();` in the catch section :)